code/modules/status_system/statusSystem.dm

```diff
@@ -755,7 +770,7 @@ var/list/statusGroupLimits = list("Food"=4)
 		onRemove()
 			..()
 			if(!owner) return
-			if (!owner.hasStatus("stunned") && !owner.hasStatus("weakened") && !owner.hasStatus("paralysis") && !owner.hasStatus("pinned")) //consider later : a way to group effects to check a bunch in one proc call and save sonme cpu
+			if (!owner.hasStatus(list("stunned", "weakened", "paralysis", "pinned"))) //consider later : a way to group effects to check a bunch in one proc call and save sonme cpu
```

Comment:
```suggestion
			if (!owner.hasStatus(list("stunned", "weakened", "paralysis", "pinned"))) //consider later : a way to group effects to check a bunch in one proc call and save sonme cpu
```
you can remove my comment because you added the thing it was asking for